lms/djangoapps/courseware/testutils.py

```diff
@@ -148,9 +148,9 @@ def verify_response(self, expected_response_code=200, url_params=None):
         return response
 
     @ddt.data(
-        ('vertical_block', ModuleStoreEnum.Type.mongo, 14),
+        ('vertical_block', ModuleStoreEnum.Type.mongo, 10),
```

Comment:
See the changes in `lms/djangoapps/courseware/date_summary.py`. Instead of making multiple calls for `CourseOverview`, `CourseMode`, and configuration models, we now make one call to the `Schedule` model.